pkgs/top-level/lua-packages.nix

```diff
@@ -150,60 +161,77 @@ let
         EXPAT_INC="-I${expat.dev}/include");
     '';
 
-    meta = {
+    disabled = isLuaJIT;
+
+    meta = with stdenv.lib; {
+      description = "SAX XML parser based on the Expat library";
       homepage = "http://matthewwild.co.uk/projects/luaexpat";
-      platforms = stdenv.lib.platforms.unix;
-      maintainers = [ stdenv.lib.maintainers.flosse ];
+      license = licenses.mit;
+      maintainers = with maintainers; [ flosse ];
+      platforms = platforms.unix;
     };
   };
 
   luafilesystem = buildLuaPackage rec {
     name = "filesystem-1.6.2";
-    src = fetchzip {
-      url = "https://github.com/keplerproject/luafilesystem/archive/v1_6_2.tar.gz";
+
+    src = fetchFromGitHub {
+      owner = "keplerproject";
+      repo = "luafilesystem";
+      rev = "v1_6_2";
       sha256 = "134azkxw84xp9g5qmzjsmcva629jm7plwcmjxkdzdg05vyd7kig1";
     };
-    preConfigure = "substituteInPlace config --replace 'CC= gcc' '';"
+
+    preConfigure = ''
+      substituteInPlace config --replace 'CC= gcc' ''';
```

Comment:
That's a bit confusing, they just meant `--replace "CC= gcc" ""`.

Reply:
Oh yeah that's simpler, good point. I was just mindlessly changing some formatting and wasn't thinking much. :see_no_evil: